internal/claims.js

```diff
@@ -1,7 +1,24 @@
-const gen = require('./genTransaction')
+const coder = require("web3-eth-abi").AbiCoder();
+const gen = require("./genTransaction");
 
 module.exports.addClaim = (contracts, _point, _protocol, _claim, _dossier) =>
-  gen.tx(contracts.claims, 'addClaim', _point, _protocol, _claim, _dossier)
+  gen.tx(contracts.claims, "addClaim", _point, _protocol, _claim, _dossier);
+
+module.exports.removeClaim = (contracts, _point, _protocol, _claim) =>
+  gen.tx(contracts.claims, "removeClaim", _point, _protocol, _claim);
+
 
 module.exports.getClaim = (contracts, whose, index) =>
-  contracts.claims.methods.claims(whose, index).call()
+  contracts.claims.methods.claims(whose, index).call();
+
+module.exports.getAllClaims = (contracts, whose) =>
+  Promise.all(
+    [...Array(16).keys()].map((i) =>
+      contracts.claims.methods.claims(whose, i).call()
+    )
+  ).then((claims) =>
+    claims.filter(
+      (claim) =>
+        claim.claim !== "" || (claim.protocol !== "" && claim.dossier) || null
```

Comment:
What's up with this check? Of course it's weird if only the protocol is filled in or w/e, but we want to surface any claims that have non-zero content in _any_ of the fields, right?

Reply:
https://github.com/urbit/azimuth/blob/master/contracts/Claims.sol#L183
I fixed it according to this. My thinking is we should surface any claim which would be passed over in findEmptySlot, or chosen in findClaim, and it checks protocol and claim but not dossier. Even if the claim is erroneous, even more so, it should be surfaced so that the bridge user may remove it.